src/python/bcc/libbcc.py

```diff
@@ -108,6 +108,12 @@
 lib.bpf_detach_tracepoint.argtypes = [ct.c_char_p, ct.c_char_p]
 lib.bpf_attach_raw_tracepoint.restype = ct.c_int
 lib.bpf_attach_raw_tracepoint.argtypes = [ct.c_int, ct.c_char_p]
+lib.bpf_detach_kfunc.restype = ct.c_int
+lib.bpf_detach_kfunc.argtypes = [ct.c_int, ct.c_char_p]
```

Comment:
Looks good. Could you remove `bpf_detach_kfunc`? In the python side, it is not needed as python just close the fd, no need to go to C++.

Reply:
Done!